scanner/updater/export.go

```diff
@@ -45,32 +45,29 @@ func Export(ctx context.Context, outputDir string, opts *ExportOptions) error {
 	if err != nil {
 		return fmt.Errorf("initializing: manual: %w", err)
 	}
-	bundles["rhel"], err = rhelOpts(ctx)
-	if err != nil {
-		return fmt.Errorf("initializing updater: rhel: %w", err)
-	}
 	bundles["nvd"] = nvdOpts()
 
 	// ClairCore updaters.
 	for _, uSet := range []string{
-		"oracle",
-		"photon",
-		"suse",
-		"aws",
 		"alpine",
+		"aws",
 		"debian",
+		"oracle",
+		"osv",
+		"photon",
 		"rhcc",
+		"rhel-vex",
+		"suse",
 		"ubuntu",
-		"osv",
 	} {
 		bundles[uSet] = []updates.ManagerOption{updates.WithEnabled([]string{uSet})}
 	}
 
 	// The http client for pulling data from security sources.
-	limiter := rate.NewLimiter(rate.Every(time.Second), 15)
 	httpClient := &http.Client{
 		Transport: &rateLimitedTransport{
-			limiter:   limiter,
+			// Rate limit to ~16 requests/second.
+			limiter:   rate.NewLimiter(rate.Every(62*time.Millisecond), 1),
```

Comment:
Fast! Any benefit to making this configurable in case we start hitting limits?

Reply:
perhaps I'll just do this outside this PR, as it's tangentially related (noticed due to this PR)

Reply:
Created https://github.com/stackrox/stackrox/pull/12774 for this